index.bs

```diff
@@ -1224,52 +1224,80 @@ as the *sender* and the agent receiving the media as the *receiver*.
 
 Capabilities {#streaming-capabilities}
 --------------------------------------------
+If the advertiser is already authenticated, the requester has the ability to
+request additional information by sending an streaming-capabilities-request
+message, and receive back a streaming-capabilities-response message with the
+following properties:
 
-<!--
-  TODO:
+: receive-audio (required)
+:: A list of audio capabilities. For an explanation of fields, see below.
```

Comment:
Perhaps "a list of capabilities for receiving audio".  Here and similar below.  Some day, we may have capabilites for sending as well, so I want to make it clear everywhere that these are receiving capabilities.

Reply:
+1